### DISCUSSION TITLE
I can't get over how beautiful this combination of colour, theme and wallpaper is

### DISCUSSION THREAD

**Discussion by TheInsane101** [Maplespe/DWMBlurGlass#404] [Show and tell]
![image](https://github.com/user-attachments/assets/9ecb98da-8feb-42ef-8037-d2342eef72e5)

**Comment by Skyyblaze** [comment]
That looks amazing, I hope we eventually get an update for 24H2! What theme is that?

**Reply by TheInsane101** [reply]
https://www.deviantart.com/vaporvance/art/Longhorn-Pearl-1051472128

**Reply by Skyyblaze** [reply]
> https://www.deviantart.com/vaporvance/art/Longhorn-Pearl-1051472128

Thank you @TheInsane101! Too bad it's not compatible with Windows 11.

**Reply by TheInsane101** [reply]
There is a script to downgrade from Windows 11 to Windows 10 2021 LTSC without losing your data. I did it on my laptop that came with Windows 11 and it worked.

Just type this command into the Run window: "powershell iex(irm ltsc.teknixstuff.com)

Sorry that this reply is so late!
